forch/test_lib/integration_base.py

```diff
@@ -32,13 +32,14 @@ def _run_forch_script(self, script, arglist=[]):
         return self._run_shell_command(command)
 
     def _setup_stack(self):
-        code, out, err = self._run_forch_script('bin/setup_stack', ['local', 'skip-conn-check'])
+        code, out, err = self._run_forch_script('bin/setup_stack',
```

Comment:
filename should be under testing still, so testing/test_lib/?

Reply:
Yup. Changed.